pom.xml

```diff
@@ -107,6 +108,13 @@
 				<version>${uiframeworkVersion}</version>
 				<scope>provided</scope>
 			</dependency>
+
+			<dependency>
```

Comment:
@reagan-meant You should also add the omod in the distro.properties file (I put that in my comments on the card)

Reply:
Done